nemo/collections/asr/modules/audio_modules.py

```diff
@@ -122,6 +140,98 @@ def num_channels(self) -> int:
                 'must be provided when constructing the object.'
             )
 
+    @staticmethod
+    def get_mean_time_channel(input: torch.Tensor, input_length: Optional[torch.Tensor] = None) -> torch.Tensor:
+        """Calculate mean across time and channel dimensions.
+
+        Args:
+            input: tensor with shape (B, C, F, T)
+            input_length: tensor with shape (B,)
+
+        Returns:
+            Mean of `input` calculated across time and channel dimension
+            with shape (B, 1, F, 1)
+        """
```

Comment:
Probably good to assert or check the input shape if you depend on specific layout of input